### ISSUE TITLE
Remove telemetry

### ISSUE THREAD

**Issue by zabil** [getgauge/gauge#1490]
Gauge does not need to keep telemetry data anymore so

- [x] Remove telemetry code from Gauge
- [x] Remove telemetry code from Gauge-JS
- [x] Remove telemetry messages
- [x] Remove the telemetry config
- [x] Remove telemetry popups from visual studio code
- [x] Remove unique id generation
- [x] Remove telemetry information from docs
- [x] Update telemetry blog.
- [ ] Archive and remove google analytics

**Comment by M4rk9696** [comment]
I'll pick this up

**Comment by nehashri** [comment]
This story is not too be picked up yet. We have left it in the backlog for now.

**Comment by M4rk9696** [comment]
Ok, let me know when this can be picked up

**Comment by nehashri** [comment]
@M4rk9696 We can start work on this. 
We need to remove the telemetry package and all its references.

**Comment by BugDiver** [comment]
Getting following error after installing latest gauge (built from source).
```
Error ----------------------------------

[Gauge]
config 'gauge_telemetry_log_enabled' doesn't exist

Get Support ----------------------------
    Docs:          https://docs.gauge.org
    Bugs:          https://github.com/getgauge/gauge/issues
    Chat:          https://gitter.im/getgauge/chat

Your Environment Information -----------
    darwin, 1.0.7, bfdb2088

```

* Install from source (master)
* run `gauge config --list`

**Comment by Apoorva-GA** [comment]
Available in commit f7e2422f6e198885c49ea62a31cb2905615ccba2
